### ISSUE TITLE
How to custom error message of RegularExpression

### ISSUE THREAD

**Issue by vimutti77** [okgrow/graphql-scalars#21]
I want to change error message of my scalar created by RegularExpression. I want to hide the regex in my production. Can you provide the way to change it easily.

In example
```
const options = {
  errMessage: (regex, value) => {
    if (process.env.NODE_ENV === 'production')
      return `Value is invalid format: ${value} `
    else
      return `Value does not match the regular expression ${regex}: ${value}`
  }
}
const MyRegexType = new RegularExpression('MyRegexType', /^ABC$/, options);
```

**Comment by RichardLitt** [comment]
Sounds like a PR which could be applied [here](https://github.com/okgrow/graphql-scalars/blob/master/src/RegularExpression.js#L5). @ccuilla Would this be interesting to add?

**Comment by ccuilla** [comment]
We'll fix this in v0.4.3 coming shortly.

**Comment by ccuilla** [comment]
Added in v0.4.3.
